packages/react-kit/src/stories/widgets/Commit.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { ComponentMeta, ComponentStory, Story } from "@storybook/react";
+
+import { CommitWidget } from "../../components/widgets/commit/CommitWidget";
+import { EnvironmentType, getEnvConfigs } from "@bosonprotocol/core-sdk";
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: "Widgets/Commit",
+  component: CommitWidget
+} as ComponentMeta<typeof CommitWidget>;
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Template: ComponentStory<typeof CommitWidget> = (args) => (
+  <CommitWidget {...args} />
+);
+
+const wrapper = (Story: Story) => (
+  <div>
+    <Story />
+  </div>
+);
+
+export const Commit: ComponentStory<typeof CommitWidget> = Template.bind({});
+
+const envName =
+  (process.env.STORYBOOK_DATA_ENV_NAME as EnvironmentType) || "testing";
+const envConfig = getEnvConfigs(envName);
+// More on args: https://storybook.js.org/docs/react/writing-stories/args
+Commit.args = {
+  envName,
+  configId: envConfig[0].configId,
+  walletConnectProjectId: process.env.REACT_APP_WALLET_CONNECT_PROJECT_ID,
+  dateFormat: "YYYY/MM/DD",
+  defaultCurrencySymbol: "$",
+  defaultCurrencyTicker: "USD",
+  contactSellerForExchangeUrl: "https://bosonapp.io/#/chat/{id}",
+  fairExchangePolicyRules:
```

Comment:
This doesn't seem to be the latest one.
Leading to consider offers created with the template ipfs://QmQ8ZTmmRV15rFaWG9KRyjFRrpaD1o2sDwZoYiWgBaAto6 as wrong 
[subgraph](https://api.thegraph.com/subgraphs/name/bosonprotocol/mumbai-testing/graphql?query=query+MyQuery+%7B%0A++productV1Products%28where%3A+%7Buuid%3A+"086b32-3fcd-00d1-0624-67513e85415c"%7D%29+%7B%0A++++variants+%7B%0A++++++offer+%7B%0A++++++++createdAt%0A++++++++metadata+%7B%0A++++++++++...+on+ProductV1MetadataEntity+%7B%0A++++++++++++id%0A++++++++++++name%0A++++++++++++exchangePolicy+%7B%0A++++++++++++++template%0A++++++++++++%7D%0A++++++++++%7D%0A++++++++%7D%0A++++++%7D%0A++++%7D%0A++%7D%0A%7D) 

![image](https://github.com/bosonprotocol/core-components/assets/7184124/193b5aad-24c8-4833-8ac5-4b6d202a320c)